web/docs/language/features.md

```diff
@@ -667,6 +667,35 @@ const Main = () => {
 export default Main
 ```
 
+### Prisma and Validation Error Helpers
+In your Actions, you may wish to handle User entity validation errors or general Prisma errors with HTTP friendly responses. We have exposed one class and two helpers functions for this purpose.
```

Comment:
Looks good! Maybe you could explicitely say what we expose `AuthError`, `isPrismaError` and `prismaErrorToHttpError`? I know it is a bit redundant, due to code blocks below, but I think it makes it more obvious what is it that we are exposing?

Reply:
Also, I wonder if we should split these into two sections, one being `prisma errors helpers`, and another `auth error`? They are not really that connected, right? We can actually then talk about `AuthError` in the part of docs where we are explaining authentication, and on the other hand talk about prisma error helpers at the place where we talk about using prisma in Operations -> yes, notice also that you said `Actions`, but it can also be in `Queries` that we get Prisma errors, so you should rather say `Operations`.

Reply:
Hey there, sure thing I can refer to them directly, split them into two sections, move auth up, and change other to ops! Will do today 👍🏻 Thanks

Reply:
Awesome, if you think that is useful let's do it :D!

Reply:
Just updated like so 🚀 Good idea, thanks
<img width="984" alt="Screenshot 2022-04-20 122314" src="https://user-images.githubusercontent.com/523636/164278858-5e4930d5-9ab5-446f-859e-533a112dad66.png">
<img width="1009" alt="Screenshot 2022-04-20 122814" src="https://user-images.githubusercontent.com/523636/164278869-8c060e8a-6cd7-4b60-8be8-8e90fa0d8867.png">